app/views/hiring_staff/vacancies/show.html.haml

```diff
@@ -1,125 +1,21 @@
-
-    %hr
-    %h2.heading-medium
-      = t('schools.about')
-      = @vacancy.school.name
-
-    %p= @vacancy.school.description
-
-    %dl.school--details
-      %dt= t('schools.address')
-      %dd= @vacancy.school.full_address
-
-      - if @vacancy.school.phase.present?
-        %dt= t('schools.phase')
-        %dd= @vacancy.school.phase&.titleize
-
-      - if @vacancy.school.school_type.present?
-        %dt= t('schools.type')
-        %dd= @vacancy.school.school_type.label
-
-      - if @vacancy.school.url.present?
-        %dt= t('schools.website')
-        %dd= @vacancy.school.url
-
-    - if @vacancy.school.geolocation
-      %div#map_zoom
-      :javascript
-        function initMap() {
-          var myLatLng = {lat: #{@vacancy.school.geolocation.x}, lng: #{@vacancy.school.geolocation.y }};
-
-          var map = new google.maps.Map(document.getElementById('map_zoom'), {
-            zoom: 17,
-            center: myLatLng
-          });
-
-          var marker = new google.maps.Marker({
-            position: myLatLng,
-            map: map,
-            title: '#{@vacancy.school.name}'
-          });
-        }
-      %script{async: true, defer: true, src: "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_MAPS_API_KEY']}&callback=initMap"}
-
-  .column-one-third
-    %aside.vacancy--metadata
-      %dl
-        %dt= t('vacancies.salary')
-        %dd= @vacancy.salary_range
-
-        %dt= t('vacancies.working_pattern')
-        %dd= @vacancy.working_pattern
-
-        - if @vacancy.part_time? && @vacancy.weekly_hours
-          %dt= t('vacancies.weekly_hours')
-          %dd= @vacancy.weekly_hours
-
-        %dt= t('vacancies.expires_on')
-        %dd= format_date(@vacancy.expires_on)
-
-        - if @vacancy.starts_on.present?
-          %dt= t('vacancies.starts_on')
-          %dd= format_date(@vacancy.starts_on)
-
-        - if @vacancy.ends_on.present?
-          %dt= t('vacancies.ends_on')
-          %dd= format_date(@vacancy.ends_on)
-
-        %dt= t('vacancies.publish_on')
-        %dd= format_date(@vacancy.publish_on)
-
-        - if @vacancy.main_subject.present?
-          %dt= t('vacancies.main_subject')
-          %dd= @vacancy.main_subject
-
-        - if @vacancy.pay_scale.present?
-          %dt= t('vacancies.pay_scale')
-          %dd= @vacancy.pay_scale
-
-        - if @vacancy.leadership.present?
-          %dt= t('vacancies.leadership_level')
-          %dd= @vacancy.leadership.title
-
-    - if @vacancy.application_link.present?
-      %br
-      = link_to t('vacancies.application_link'), @vacancy.application_link, target: '_blank', class: 'button'
+    - if @vacancy.publish_today?
+      %p
+        = t('vacancies.view_posted_vacancy')
+      %p
+        = link_to vacancy_url(@vacancy), vacancy_path(@vacancy)
+    - else
+      %p
+        = t('vacancies.preview_posted_vacancy', date: @vacancy.publish_on)
+      %p
+        = link_to vacancy_url(@vacancy), vacancy_path(@vacancy)
```

Comment:
Linking back to the public vacancy view may result in a dead end if it's not yet published. Could we duplicate the show view on this page with partials and also keep hiring staff within the logged in domain and to make it easy to create another vacancy?

In either case the language here needs a tweak:

> The vacancy will be posted on 20 October 2019, you can preview it here:

You can never use that link as a preview, we could say that it will become available at this link?

Reply:
@tahb regarding this, i couldnt see any issues with the text, as `vacanies.preview_posted_vacancy` is mapped corretly in the localed, but i have moved show -> summary and added a schools/vacancy/show. Which reuses a common partial with the existing unauthorised vacancy view